nilearn/reporting/_get_clusters_table.py

```diff
@@ -267,7 +272,19 @@ def get_clusters_table(stat_img, stat_threshold, cluster_threshold=None,
                            Rows corresponding to subpeaks will not have a value
                            in this column.
         ================== ====================================================
+        
+    label_maps : :obj:`list`
+        Returned if return_label_maps=True
+        List of Niimg-like objects of cluster label maps.
+        If two_sided==True, first and second maps correspond to positive and negative tails.
```

Comment:
I think it makes sense to have one of these for the return value also:

```suggestion
        If two_sided==True, first and second maps correspond to positive and negative tails.
        
        .. versionadded:: 0.10.1.dev
        
```